tagging/utils.py

```diff
@@ -3,12 +3,25 @@
 calculation.
 """
 import math
-import types
 
 from django.db.models.query import QuerySet
-from django.utils.encoding import force_unicode
+try:
+    from django.utils.encoding import force_unicode
+except ImportError:
+    def force_unicode(s, *args, **kwargs):
+        return s
 from django.utils.translation import ugettext as _
 
+try:
+    basestring
+except NameError:  # Python 3 doesn't have basestring
+    basestring = str
+
+try:
+    long
+except NameError:  # Python 3 doesn't have long
+    long = int
```

Comment:
`builtins` is part of the future library. So I think I am going to keep it my way. Unless you think it's worth adding a dependency to this library.